book/src/configuration.md

```diff
@@ -79,6 +79,24 @@ interested in:
 cargo clippy -- -A clippy::all -W clippy::useless_format -W clippy::...
 ```
 
+The last way to allow/disallow lints is to use `Cargo.toml` using [The lints section](`Cargo.toml` using [The lints section](`Cargo.toml` using [The lints section](https://github.com/rust-lang/rust-clippy/blob/master/book/src/configuration.md#allowingdenying-lints)):
```

Comment:
TBH all this looks like a LLM fuck-up :sweat_smile: 

Please just link to the section in the cargo book and add at most one concrete (!) example.

Reply:
no all me, chatgpt explanation was way too long, couldn't use it